lib/assessment.md

```diff
@@ -163,5 +172,6 @@ Consider the following media queries. How wide would the screen need to be for e
 ```
 
 <!-- your answer starts here -->
-
+600px would make 150px 25% of the screen.
+450px would make 150px 30% of the screen.
```

Comment:
But the rule that makes width equal to 30% is inside a media query, and that media query only exposes its CSS when the screen width is 800px or wider. Therefore, at a screen width of 450px, the rule indicates that `.stretch` elements should have a width of 112.5px.